src/config.rs

```diff
@@ -43,6 +43,8 @@ pub struct ServerConfig {
     pub max_content_length: Byte,
     /// Storage path.
     pub upload_path: PathBuf,
+    /// Maximum upload directory size.
+    pub max_uploads: Option<Byte>,
```

Comment:
I would use a more distinct name. `max_uploads` sounds like how many uploads one can make.

Why don't use:

```suggestion
    pub max_upload_dir_size: Option<Byte>,
```

Reply:
Yup, this change makes sense to me 👍🏼

Reply:
Ah, I aee that you actually used my suggested name before. Cool. Thanks for switching it back.